src/aliceVision/sfm/bundle/BundleAdjustmentCeres.cpp

```diff
@@ -909,7 +1021,7 @@ bool BundleAdjustmentCeres::adjust(sfmData::SfMData& sfmData, ERefineOptions ref
     ceres::Solve(options, &problem, &summary);
 
     // print summary
-    if (_ceresOptions.summary)
+    //if (_ceresOptions.summary)
```

Comment:
Why is the condition commented if the option still exists?
```suggestion
    if (_ceresOptions.summary)
```

Reply:
Forgot this one, i commented out for testing purposes